### PR TITLE
Add customisable apache entry point

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by asaunier** [comment]
Is it really customizable, since both files are CONST files?

**Comment by sbrunner** [comment]
Yes, You add `APACHE_ENTRY_POINT = /` in your Makefile and because we have a `?= ` instance of `=` the value will not be overwritten :-)

**Comment by asaunier** [comment]
OK
Is is documented somewhere?

**Comment by sbrunner** [comment]
Good point :-)

**Comment by sbrunner** [comment]
Updated :-)

File: doc/integrator/make.rst [hunk 1]
```diff
@@ -64,6 +64,8 @@ The following variables may be set in the makefiles:
 * ``PRE_RULES``: predefine some build rules, default is empty.
 * ``POST_RULES``: postdefine some build rules, default is empty.
 * ``DISABLE_BUILD_RULES``: List of rules we want to disable, default is empty.
+* ``INSTANCE_ID``: The WSGI instance id (should bu unique on a server).
```
Comment (asaunier) [inline doc/integrator/make.rst]: bu => be

**Comment by asaunier** [comment]
Small typo. Else OK for me.

**Pull request merged**